README.md

```diff
@@ -433,6 +433,29 @@ type UseName = {
 };
 ```
 
+### Avatar
+
+The `Avatar` component is used to display ENS avatar associated with Ethereum addresses. When an ENS avatar is not available, it defaults to blue color avatar.
+
+```tsx
+import { Avatar } from '@coinbase/onchainkit';
+
+<Avatar address="0x1234567890abcdef1234567890abcdef12345678" />;
+```
+
+**@Props**
+
+```ts
+type UseAvatar = {
```

Comment:
nit for follow up PR, I believe `UseAvatar` should just be `Avatar`.

Also we should start get in the habit to showcase the return html content. under props